cmd/rook/main.go

```diff
@@ -232,6 +240,14 @@ func setLogLevel() {
 	capnslog.SetGlobalLogLevel(cfg.logLevel)
 }
 
+func logStartupInfo(cmdFlags *pflag.FlagSet) {
+	// log the version number, arguments, and all final flag values (environment variable overrides
+	// have already been taken into account)
+	flagValues := flags.GetFlagsAndValues(cmdFlags, "secret")
```

Comment:
haha OK, i'd like to keep the implementation as is that strictly meets the requirements for the flags we currently have defined.  Let me know if there are other changes you need to approve this PR.